ecommerce/extensions/payment/processors/paystack.py

```diff
@@ -181,7 +191,11 @@ def issue_credit(self, order_number, basket, reference_number, amount, currency)
             RefundError: indicating general refund error.
         """
         try:
-            success, response = self.paystack_client.handler(CREATE_REFUND_CODE, reference_number)
+            api_data = {
```

Comment:
if we do not send any amount in refund request then Paystack reverses the whole transaction amount back. To fix partial refund issue, we need to send the amount in request data.